recipe/build.sh

```diff
@@ -24,11 +9,14 @@ else
   extra_args=""
 fi
 
+export SETUPTOOLS_SCM_PRETEND_VERSION_FOR_CIL="${PKG_VERSION}"
```

Comment:
`SETUPTOOLS_SCM_PRETEND_VERSION_FOR_CIL`

What is this for? and is `PKG_VERSION` set somewhere or is it a manual step? Currently it gets the version from git, I think historically `CIL_VERSION` had to be set - and maybe still is in some other repos.